src/fmu/dataio/dataio.py

```diff
@@ -39,6 +39,9 @@
 from .datastructure.configuration import global_configuration
 from .providers._fmu import FmuEnv
 
+# always show PendingDeprecationWarnings
+warnings.simplefilter("always", PendingDeprecationWarning)
```

Comment:
Had to include this to make warnings actually show, not sure if this is specific for me 🤷‍♂️

Reply:
Later learned that we should use other types of warnings to make sure they are actually seen by end users.

Reply:
Will remove this, but keeping it for now as a TODO: We need to make sure warnings are actually seen by end users/modellers. Possibly separate PR.